docs/cli-commands.mdx

```diff
@@ -696,6 +696,45 @@ Sets the name for the chain. Default: `example`.
 
 ---
 
+<h4><i>pos</i></h4>
+
+<Tabs>
+  <TabItem value="syntax" label="Syntax" default>
+
+    genesis [--pos IS_POS]
```

Comment:
Do you need `IS_POS`?

Reply:
It's optional of course, I just followed the same form we used for the `--seal` flag to keep it uniform 👍 

https://github.com/0xPolygon/polygon-sdk-docs/blob/main/docs/cli-commands.mdx#seal